src/RulesFactory.js

```diff
@@ -0,0 +1,69 @@
+import warning from 'warning'
+
+import RegularRule from './rules/RegularRule'
+import SimpleRule from './rules/SimpleRule'
+import KeyframeRule from './rules/KeyframeRule'
+import ConditionalRule from './rules/ConditionalRule'
+import FontFaceRule from './rules/FontFaceRule'
+
+export default class RulesFactory {
+  constructor() {
+    this.classes = {
```

Comment:
If this is going to be set in the constructor, it should be extendable via an options object. No point otherwise.